src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseFVM.cpp

```diff
@@ -110,30 +110,32 @@ void CompositionalMultiphaseFVM::assembleFluxTerms( real64 const dt,
    *       Or stop testing through the solver interface and test separate kernels instead.
    *       Finally, the problem should go away when fluid updates are executed on device.
    */
-  forTargetSubRegions( mesh, [&]( localIndex const targetIndex, ElementSubRegionBase const & subRegion )
-  {
-    MultiFluidBase const & fluid = getConstitutiveModel< MultiFluidBase >( subRegion, fluidModelNames()[targetIndex] );
-    arrayView4d< real64 const > const & phaseCompFrac = fluid.phaseCompFraction();
-    arrayView4d< real64 const > const & dPhaseCompFrac_dPres = fluid.dPhaseCompFraction_dPressure();
-    arrayView5d< real64 const > const & dPhaseCompFrac_dComp = fluid.dPhaseCompFraction_dGlobalCompFraction();
-
-    arrayView3d< real64 const > const & phaseMassDens = fluid.phaseMassDensity();
-    arrayView3d< real64 const > const & dPhaseMassDens_dPres = fluid.dPhaseMassDensity_dPressure();
-    arrayView4d< real64 const > const & dPhaseMassDens_dComp = fluid.dPhaseMassDensity_dGlobalCompFraction();
-
-    forAll< parallelDevicePolicy<> >( subRegion.size(),
+  /*
+     forTargetSubRegions( mesh, [&]( localIndex const targetIndex, ElementSubRegionBase const & subRegion )
+     {
+     MultiFluidBase const & fluid = getConstitutiveModel< MultiFluidBase >( subRegion, fluidModelNames()[targetIndex] );
+     arrayView4d< real64 const > const & phaseCompFrac = fluid.phaseCompFraction();
+     arrayView4d< real64 const > const & dPhaseCompFrac_dPres = fluid.dPhaseCompFraction_dPressure();
+     arrayView5d< real64 const > const & dPhaseCompFrac_dComp = fluid.dPhaseCompFraction_dGlobalCompFraction();
+
+     arrayView3d< real64 const > const & phaseMassDens = fluid.phaseMassDensity();
+     arrayView3d< real64 const > const & dPhaseMassDens_dPres = fluid.dPhaseMassDensity_dPressure();
+     arrayView4d< real64 const > const & dPhaseMassDens_dComp = fluid.dPhaseMassDensity_dGlobalCompFraction();
+
+     forAll< parallelDevicePolicy<> >( subRegion.size(),
```

Comment:
remove (we have `git`)?

Reply:
@TotoGaz  Thanks for the comments, I am going to address all of them next week. 

The modification above is temporary for my verification tests and the tests on Pangea3: since the fluid updates are done on device for CO2-brine, we do not need this piece of code that is manually moving to the GPU some arrays used in the flux computations (`phaseCompFrac` et `phaseMassDens`). As soon as we have the Pangea3 timings and we are ready to merge this PR, this will need to be uncommented back (integrated tests failures will remind us to uncomment this).

Reply:
I uncommented this piece of code (we still need it for non-GPU fluid models)